### DISCUSSION TITLE
CHGnet predicted energy

### DISCUSSION THREAD

**Discussion by Yong-Q** [CederGroupHub/chgnet#93] [Q&A]
Hi, I don't know if the energy predicted by CHGnet is toal energy, I compare the energy obtained by Vasp (free energy TOTEN) to it and it gives an error of 10ev, the number of atoms is 48.
And I don't know which of the MP terms the predicted energy comes from
Thank you for your help. It's very important for me to understand.

**Comment by bowen-bd** [comment]
Please have a look on [MPtrj](https://figshare.com/articles/dataset/Materials_Project_Trjectory_MPtrj_Dataset/23713842) dataset description
